webapp/src/App.js

```diff
@@ -171,13 +172,13 @@ class App extends Component {
                       >
                         Review Assignment
                       </NavLink>
-                      <NavLink
```

Comment:
Was this removed on purpose? :)

Reply:
@mthembbu please see Kale-ab's comment here - we don't think this item should have been removed.

Reply:
I don't recall making the changes. Will be placing the lines back. Thanks a lot for making me be aware.